plugin/RunalyzePluginPanel_TagsSummary/class.RunalyzePluginPanel_TagsSummary.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * This file contains the class of the RunalyzePluginPanel "Tags Summary".
+ * @package Runalyze\Plugins\Panels
+ */
+$PLUGINKEY = 'RunalyzePluginPanel_TagsSummary';
+
+use Runalyze\Model;
+
+/**
+ * Class: RunalyzePluginPanel_TagsSummary
+ *
+ * @author Felix Gertz
+ * @package Runalyze\Plugins\Panels
+ */
+class RunalyzePluginPanel_TagsSummary extends PluginPanel {
+	/**
+	 * Internal array with all tags from database
+	 * @var array
+	 */
+	private $Tags = null;
+
+	/**
+	 * @var array
+	 */
+	protected $AllSports = array();
+
+	/**
+	 * Name
+	 * @return string
+	 */
+	final public function name() {
+		return __('Tags summary');
+	}
+
+	/**
+	 * Description
+	 * @return string
+	 */
+	final public function description() {
+		return __('Summarises your tags for the selected sport.');
+	}
+
+	/**
+	 * Init configuration
+	 */
+	protected function initConfiguration() {
+		$this->AllSports = DB::getInstance()
```

Comment:
This can be simplified by using the static factory (which uses a cache containing all sport types):
```php
foreach (\Runalyze\Context::Factory()->allSports() as $sport) {
    $this->AllSports[$data['id']] = $sport->name();
}

$Options = $this->AllSports;
```